packages/manager/modules/octavia-load-balancer/src/delete/delete.routing.js

```diff
@@ -0,0 +1,39 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.delete', {
+    url: '/delete?loadBalancerId&loadBalancerName&loadBalancerRegion',
+    views: {
+      modal: {
+        component: 'octaviaLoadBalancerDelete',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null,
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().projectId,
+      loadBalancerId: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerId,
+      loadBalancerRegion: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerRegion,
+      loadBalancerName: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerName,
+      goBack: /* @ngInject */ ($state) => () =>
+        $state.go('octavia-load-balancer'),
+      returnToListingSuccess: /* @ngInject */ ($state) => () =>
+        $state.go(
+          'octavia-load-balancer',
+          {
+            loadBalancerDeleteSuccess: true,
```

Comment:
no need this param. See https://github.com/ovh/manager/pull/10044/files#r1343787678

Reply:
Done